pkg/repositories/repositories.go

```diff
@@ -57,6 +57,44 @@ func NewConfig(filename string) (Config, error) {
 	return repoConfig, nil
 }
 
+// If our first config and our second config share a repository,
+// we set the afterVersion of the first to its original version, and the
+// version of the first to the version of the second.
+func CombineConfigs(currentConfig *Config, newConfig *Config) {
+
```

Comment:
No need for this extra newline

Reply:
You know how I hate those newlines ;) 
Good catch - thanks